md/bonita-bpm-portal-interface-overview.md

```diff
@@ -1,32 +1,49 @@
-# Bonita Portal overview
+# Bonita User Interfaces overview
 
-Bonita Portal allows you to access server administration and monitoring for administrators, and a generic process view for users.
+This page describes the User Interfaces provided by Bonita.   
+They rely on the analysis that four types of personas use process-based applications in a company:
+  - the **Administrator** is responsible for the administration of Bonita at tenant level, and particularly for the management of the processes, the organization, the reports, the custom profiles and the Look & Feel.
+  - the **Process Manager** shares process management responsibilities with the Administrator for the processes he has been declared as the Process Manager.
+  - the **User** is responsible of performing the tasks for which s·he is a candidate and also for starting new cases of the processes to which s·he has access.
+  - the **Technical User** is responsible for opening permissions to use Bonita in non-production or production environments by creating users. S·he is also the only person with permissions to start and stop the BPM services for maintenance or update (namely BDM update). For more information, go to the dedicated [documentation page](first-steps-after-setup.md).
```

Comment:
Probably use the pause-and-resume-bpm-services.md and bdm-management-in-bonita-bpm-portal.md

Reply:
this is not up to date